quippy/quippy/potential.py

```diff
@@ -44,24 +44,26 @@ class Potential(ase.calculators.calculator.Calculator):
     callback_map = {}
 
     implemented_properties = ['energy', 'free_energy', 'forces', 'virial', 'stress',
-                              'local_virial', 'local_energy', 'local_stress']
+                              'local_virial', 'local_energy', 'local_stress', 'stresses', 'energies']
```

Comment:
We could consider if we should remove `local_stress` altogether - `stresses` is the expected ASE-compatible name.

Reply:
ok